src/main/java/LeaveGroup/LeaveGroupResponseModel.java

```diff
@@ -0,0 +1,31 @@
+package LeaveGroup;
```

Comment:
I don't think the response model can interact with the entities?

Reply:
omg yeah you're right. Oops, I misunderstood the response model, I think. I've removed the entitites and am using Strings now. Thank you!